python/cuml/test/test_dbscan.py

```diff
@@ -48,10 +51,12 @@ def test_dbscan_predict(datatype, input_type):
 
     for i in range(X.shape[0]):
         assert cu_labels[i] == sk_labels[i]
+    cudbscan.sync()
```

Comment:
If I am not missing anything the stream sync is needed before the assert loop in line 52-53.

Reply:
Done. Thanks for catching this.